styles/PageTitle.module.css

```diff
@@ -0,0 +1,48 @@
+.container {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    flex-wrap: nowrap;
+    margin-bottom: 50px;
+    cursor: pointer;
+    width: 100%;
+}
+
+.titleBox {
+    width: 25%;
+    padding: 0 20px 0 20px;
```

Comment:
Minor styling change, but I would change to right padding of 30px, and then change the `h1` text-align to be `right` instead of `center`. In the designs the title is more at the end of the blue border, not the middle.

Reply:
![image](https://user-images.githubusercontent.com/35358725/99312970-1a1c5280-285f-11eb-8c18-57ba44a496c8.png)

![image](https://user-images.githubusercontent.com/35358725/99313017-256f7e00-285f-11eb-811f-f0c81eec1b0c.png)

To be honest I am not really loving the title? I had the blue box extended to the end and it looked off. Do you like it at the end better?